torch/library.py

```diff
@@ -327,6 +327,19 @@ def impl(self, op_name, fn, dispatch_key="", *, with_keyset=False):
         _impls.add(key)
         self._op_impls.add(key)
 
+    def fallback(self, fn, dispatch_key="", *, with_keyset=False):
```

Comment:
Do we care about public API for this?

Reply:
Do you mean that you want me to make this private?
In the spirit of torch.library.Library can do everything the C++ one can do, this one next to .impl() makes sense as a public API to me.

But happy to move it to private if you feel strongly about it.

Reply:
Oh I meant if you wanted to add a docstring

Reply:
Oups!